<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPaciente</class>
 <widget class="QWidget" name="TelaPaciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_titulo">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Passo 2: Dados do Paciente</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Informações do paciente</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_nome">
        <property name="text">
         <string>Nome Completo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_nome"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_nascimento">
        <property name="text">
         <string>Data de Nascimento:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEdit_nascimento">
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_idade_crono">
        <property name="text">
         <string>Idade Cronológica:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_idade_crono"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_escola">
        <property name="text">
         <string>Instituição de Ensino:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_escola"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_turma">
        <property name="text">
         <string>Turma:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_turma"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Informações dos responsáveis</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_resp1_nome">
           <property name="text">
            <string>1° Responsável:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_resp1_escolaridade">
           <property name="text">
            <string>Escolaridade:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_resp1_nome"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_resp1_escolaridade"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_resp1_idade">
           <property name="text">
            <string>Idade:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_resp1_profissao">
           <property name="text">
            <string>Profissão:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_resp1_idade"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_resp1_profissao"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_resp2_nome">
           <property name="text">
            <string>2° Responsável:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_resp2_escolaridade">
           <property name="text">
            <string>Escolaridade:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_resp2_profissao">
           <property name="text">
            <string>Profissão:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_resp2_idade">
           <property name="text">
            <string>Idade:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_resp2_nome"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_resp2_idade"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_resp2_profissao"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_resp2_escolaridade"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Informações da solicitante</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_solicitante_nome">
        <property name="text">
         <string>Nome Completo:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_solicitante_crp">
        <property name="text">
         <string>CRP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_solicitante_nome"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_solicitante_crp"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_voltar">
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_avancar">
       <property name="text">
        <string>Avançar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
